### PR TITLE
Update templates to use scoped CSS

### PR DESCRIPTION
**Pull request by captainsafia**
Address https://github.com/dotnet/aspnetcore/issues/31030

### CODE REVIEW

File: src/ProjectTemplates/test/template-baselines.json [hunk 1]
```diff
@@ -22,11 +22,11 @@
         "Pages/_ViewImports.cshtml",
         "Pages/_ViewStart.cshtml",
         "Pages/Shared/_Layout.cshtml",
+        "Pages/Shared/_Layout.cshtml.css",
         "Pages/Shared/_LoginPartial.cshtml",
         "Pages/Shared/_ValidationScriptsPartial.cshtml",
         "Properties/launchSettings.json",
         "wwwroot/favicon.ico",
-        "wwwroot/css/site.css",
```
Comment (captainsafia) [inline src/ProjectTemplates/test/template-baselines.json:29]: Do we want to keep a stub `site.css` file in the template or should we completely move over?

Comment (javiercn) [reply src/ProjectTemplates/test/template-baselines.json:29]: I think we should keep it, it gives folks a place to put regular/global CSS.

Comment (pranavkm) [reply src/ProjectTemplates/test/template-baselines.json:29]: We generally don't ship empty files though. Particularly not one that isn't referenced by the app. Let's remove this, users can always add one back if they want to.

**Review by pranavkm — APPROVED** [review]

**Review by pranavkm — APPROVED** [review]

**Comment by pranavkm** [comment]
@captainsafia do we plan on merging this in to main too?

**Comment by captainsafia** [comment]
@pranavkm Yep.

**Comment by captainsafia** [comment]
@dotnet/aspnet-build Can I get help merging?

**Comment by captainsafia** [comment]
@dotnet/aspnet-build Can I get help merging this?

**Pull request merged**